### PR TITLE
CU-860payu3p_Develop-Cards-Component_Fadhlan-Hasyim

### PR DESCRIPTION
**Pull request by fadhlanhasyim**

### CODE REVIEW

**Review by gibranoderenz — APPROVED** [review]
thanks dlan mantap

File: src/Cards/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { CardsProps } from './interface'
+import { Content, Title, Body, Main, Image } from './Cards.style'
+
+const Cards: React.FC<CardsProps> = ({ ...props }) => {
+  const { imageUrl, title, body, className, imgClassName, width, height } =
```
Comment (DJaegerScript) [inline src/Cards/index.tsx]: destructurenya dalam 1 line aja

File: src/Cards/index.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    props
+
+  return (
+    <Main id="main" className={className} width={width} height={height}>
```
Comment (DJaegerScript) [inline src/Cards/index.tsx]: `id` nya dijadiin props

File: src/Cards/interface.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export interface CardsProps {
+  imageUrl?: string
+  title?: string
+  body?: string
+  className?: string
+  imgClassName?: string
+  width: number
+  height: number
```
Comment (DJaegerScript) [inline src/Cards/interface.ts]: kenapa harus aja ini?

mending dibikin size variant aja

**Review by DJaegerScript — CHANGES_REQUESTED** [review]

**Review by DJaegerScript — CHANGES_REQUESTED** [review]
- use `isExpand` props rather than `getElementById` to change styling
- add popup animation

**Review by DJaegerScript — APPROVED** [review]

**Pull request merged**